packages/ui/src/components/DataStory/Form/StringableInput.tsx

```diff
@@ -1,68 +1,66 @@
-import { StringableParam, get } from '@data-story/core';
-import { useContext, useEffect, useState } from 'react';
-import { FormFieldWrapper, FormFieldContext, useFormField } from './UseFormField';
-
-// Function to calculate the number of rows based on content
-const calculateRows = (content: string) => {
-  const newLines = content.split('\n').length;
-  return Math.max(newLines, 1); // Ensure a minimum of 1 row
-};
+import {  StringableParam } from '@data-story/core';
+import {  useCallback} from 'react';
+import { FormFieldWrapper, useFormField } from './UseFormField';
+import { ControllerRenderProps } from 'react-hook-form';
+import { autocompletion } from '@codemirror/autocomplete';
+import CodeMirror, { BasicSetupOptions } from '@uiw/react-codemirror';
 
 interface StringableInput {
   param: StringableParam;
   onCursorPositionChange: (position: number) => void; // Add this line
+  field?: ControllerRenderProps<any, `${string}.value`>;
```

Comment:
What is the purpose of this field? Might be mistaken but I dont see it is being used.